kubernetes/kubernetes.go

```diff
@@ -120,8 +138,9 @@ func CollectKubeMetrics(config KubeAgentConfig) {
 	err := fetchDiagnostics(kubeAgent.Clientset, config.Namespace, kubeAgent.msExportDirectory)
 
 	if err != nil {
+		log.Warnf(rbacError, kbProvisionURL)
```

Comment:
⭐ 
I think this could be clearer if `rbacError` was updated to include the provisioning url directly. My thinking is, the `rbacError` will always need to include the link to docs on provisioning which gets rid of the need to add functionality to support another string with the `%s` (in the `rbacError` string). We could always add that support in later if we need it. I'd imagine the error message would need to be tweaked if that's the case.

To illustrate what I'm thinking:
```
const rbacError string = `RBAC role in the Cloudability namespace may need to be updated.
	Re-provision the metrics-agent for this cluster as detailed here: ` + kbProvisionURL + [rest of string]
```